linode_api4/linode_client.py

```diff
@@ -871,6 +871,213 @@ def users(self, *filters):
         """
         return self.client._get_and_filter(User, *filters)
 
+    def logins(self):
+        """
+        Returns a collection of successful logins for all users on the account during the last 90 days.
+
+        API Documentation: https://www.linode.com/docs/api/account/#user-logins-list-all
+
+        :returns: A list of Logins on this account.
+        :rtype: PaginatedList of Login
```

Comment:
I think `:rtype:` should be the strict Python type, like the name of a class, right?
It's for auto docs generation.

Reply:
Oh I didn't realize that it was for auto generated docs. I will fix this!